opencti-platform/opencti-front/src/private/components/settings/users/UserConfidenceLevel.tsx

```diff
@@ -55,11 +52,18 @@ const UserConfidenceLevel: React.FC<UserConfidenceLevelProps> = ({ confidenceLev
           </Tooltip>
         );
       }
-      // the user himself
+
+      // source is the user himself, most probably by setting at the user level
+      let title = t_i18n('The Max Confidence Level is currently defined at the user level. It overrides Max Confidence Level from user\'s groups.');
+      // ... or if user has BYPASS, it's fixed to 100; check below is a cheap proxy (as we do not have user's capabilities in UserEditionOverview
+      if (!user.user_confidence_level || user.user_confidence_level.max_confidence < 100) {
```

Comment:
having a precise check on `user.capabilities` won't work immediately within `UserEditionOverview`. 

One thing leading to another, it requires returning capabilities from the field patch mutation, which is not made that way (it only returns unresolved object). I'm not inclined to modify this behavior and call `buildCompleteUser` after `updateAttribute`. Seems too costly and deviating from our initial strategy (specific mutations corresponding to resolved fields).